client/components/organization-wrapper/organization-wrapper.js

```diff
@@ -11,16 +11,17 @@ import ConnectedDoesNotExist from "../404";
 import DoesNotExist from "../404/404";
 import Footer from "../footer";
 import Header from "../header";
-import Login from "../login";
-import PasswordConfirm from "../password-confirm";
-import PasswordReset from "../password-reset";
-import Registration from "../registration";
-import MobilePhoneVerification from "../mobile-phone-verification";
-import Status from "../status";
-import PasswordChange from "../password-change";
-import MobilePhoneChange from "../mobile-phone-change";
 import LoadingContext from "../../utils/loading-context";
+import Loader from "../../utils/loader";
 
+const Login = React.lazy(() => import('../login'));
+const Registration = React.lazy(() => import('../registration'));
+const PasswordChange = React.lazy(() => import('../password-change'));
+const MobilePhoneChange = React.lazy(() => import('../mobile-phone-verification'));
```

Comment:
this is importing the wrong file, should import `../mobile-phone-change`.
This broken line was causing that component to break, and our tests do not catch this because our tests import the file directly, so please watch out!